support/faq/using-sink-server.md

```diff
@@ -5,12 +5,14 @@ description: "Sink testing allows you to send messages through Spark Post for in
 
 Sink testing allows you to send messages through SparkPost for integration testing without the messages being attempted for delivery to actual email addresses. Any messages using the sink server email address format will automatically fail as intended.
 
-To use the sink server, you will need to append the email addresses you are injecting into our system with `*.sink.sparkpostmail.com`. Please note, the asterisk (`*`) is a wildcard, where you will place the domain portion of your injected email address.
+To use the sink server, you will need to append the email addresses you are injecting into our system with `*.sink.sparkpostmail.com`. Please note, the asterisk (`*`) is a wildcard, where you will place the domain portion of your injected email address. You also need to prepend the campaign_id you are injecting into our system with `sparkpost-performance-test-*`. Please note, the asterisk (`*`) is a wildcard, where you will place anything as part of the campaign_id.
```

Comment:
Can we either make this a list like:
```
To use the sink server, you will need to
1. append the...
2. prepend the...
```
or something similar.  The way it is right now, if somebody stops reading after that first sentence, they will miss the new instructions.

Also, maybe we should use the word 'prefix' instead of 'prepend'?  I think the word 'prepend' is kind of obscure.  So something like "You also need to prefix the campaign id" OR "You also need to apply the `sparkpost-performance-test-*` prefix to the campaign_id"